examples/connext_dds/waitset_status_cond/py/USER_QOS_PROFILES.xml

```diff
@@ -9,59 +9,54 @@
  any incidental or consequential damages arising out of the use or inability to
  use the software.
  -->
+<!-- Description XML QoS Profile for waitsets The QoS configuration of the
+    DDS entities in the generated example is loaded from this file. This file
+    is used only when it is in the current working directory or when the enviroment
+    variable NDDS_QOS_PROFILES is defined and points to this file. For more information
+    about XML QoS Profiles see Chapter 15 in the RTI Connext user manual. -->
+
 <dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-     xsi:noNamespaceSchemaLocation="file:////home/luisrg/Downloads/rti_connext_dds-7.4.0/resource/schema/rti_dds_qos_profiles.xsd">
-    <!-- QoS Library containing the QoS profile used in the generated example.
+     xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/7.4.0/rti_dds_qos_profiles.xsd">
 
+    <!-- QoS Library containing the QoS profile used in the generated example
         A QoS library is a named set of QoS profiles.
     -->
-    <qos_library name="waitset_status_cond_Library">
-
-        <!-- QoS profile used to configure reliable communication between the DataWriter 
+    <qos_library name="waitset_status_cond_py_Library">
+        <!-- QoS profile used to configure reliable communication between the DataWriter
              and DataReader created in the example code.
-
              A QoS profile groups a set of related QoS.
         -->
-        <qos_profile name="waitset_status_cond_Profile" base_name="BuiltinQosLib::Generic.StrictReliable">
-            <!-- QoS used to configure the data writer created in the example code -->                
+        <qos_profile name="waitset_status_cond_py_Profile" base_name="BuiltinQosLibExp::Generic.StrictReliable" is_default_qos="true">
```

Comment:
The experiemental library is only maintained for backward compatibility, as far as I know. The reliability and history set below are redundant as well.
```suggestion
        <qos_profile name="waitset_status_cond_py_Profile" base_name="BuiltinQosLib::Generic.StrictReliable" is_default_qos="true">
```

Reply:
I'll remove the `Exp`. I basically took it from the Modern C++ equivalent for simplicity:

https://github.com/rticommunity/rticonnextdds-examples/blob/master/examples/connext_dds/waitset_status_cond/c%2B%2B11/USER_QOS_PROFILES.xml

I suppose we can delete the `datawriter_qos`/`datareader_qos` part of the `.xml`, is that what you mean?